### DISCUSSION TITLE
How to add CustomItems in cart?

### DISCUSSION THREAD

**Discussion by mohasinakonda** [bigcommerce/catalyst#672] [General]
I am trying to create project with catalyst, here I wanted to add customItems in cart,
but I am not getting anything about adding customItems

**Comment by bookernath** [comment]
Hi @mohasinakonda ,

The creation of custom items is not available in Storefront GraphQL API by design. You must use the [V3 Cart API](https://developer.bigcommerce.com/docs/rest-management/carts/items#add-cart-line-items) for this as it is a sensitive action. You may call the V3 API within your Catalyst storefront for this if you wish.
